parlai/scripts/eval_model.py

```diff
@@ -52,15 +52,6 @@ def setup_args(parser=None):
     parser.add_argument('-ne', '--num-examples', type=int, default=-1)
     parser.add_argument('-d', '--display-examples', type='bool', default=False)
     parser.add_argument('-ltim', '--log-every-n-secs', type=float, default=2)
-    parser.add_argument(
```

Comment:
Hmm, it's a bit unclear from `aggregate_named_reports()` alone, but I assume that it's doing micro-averaging since that's the point of tracking both the numerator and denominator? In any case, removing this flag indicates to me that it's clearly not being used much, ha

Reply:
Yeah we're explicitly doing micro averaging always now (correctly). @emilydinan are you okay with that?